app/views/content/help-and-support/_categories.html.erb

```diff
@@ -0,0 +1,25 @@
+<section class="category">
+  <section class="container category__cards main-section">
+    <h2 class="heading--box-blue">Talk to an adviser</h2>
+    <div class="inset">
+      <p>Get dedicated one-to-one support from an adviser with years of teaching experience.</p>
+    </div>
+
+    <nav class="category__nav-cards">
+      <ul class="inset">
+        <%= render Categories::CardComponent.new(card:
+          OpenStruct.new(
+            title: "Explore teaching advisers",
```

Comment:
These adviser cards are appearing as h2s @ethax-ross - should be h3s as they are under the 'Talk to an adviser' heading which is an h2